pkg/gangway/gangway.go

```diff
@@ -349,6 +490,17 @@ func (cjer *CreateJobExecutionRequest) Validate() error {
 	return nil
 }
 
+func (bjscr *BulkJobStatusChangeRequest) Validate() error {
```

Comment:
Do we need to validate that the startedBefore and startedAfter are parseable as times if they are provided?